databroker/core.py

```diff
@@ -301,9 +301,10 @@ def documents_to_xarray(*, start_doc, stop_doc, descriptor_docs,
             continue
         if any(data_keys[key].get('external') for key in keys):
             filler('descriptor', descriptor)
+            filled_events = []
             for event in events:
                 try:
-                    filler('event', event)
+                    filled_events.append(filler('event', event)[1])
```

Comment:
Why do we append the element `[1]` from the filler?

Reply:
`filler` returns a `name, doc` pair. We want the doc.

Reply:
Great, thanks for the explanation. Do you think it would be useful to put exactly this comment into the code for the convenience of a random reader not encountered this thread?

Reply:
updated